src/utils/image.ts

```diff
@@ -125,15 +129,17 @@ export function getItemTypeIcon(itemType: BaseItemKind | string) {
         case BaseItemKind.Folder:
             return 'folder';
         case BaseItemKind.BoxSet:
-            return 'collections';
+            return 'video_library';
         case BaseItemKind.Playlist:
-            return 'view_list';
+            return 'queue';
         case BaseItemKind.Photo:
             return 'photo';
         case BaseItemKind.PhotoAlbum:
             return 'photo_album';
+        case undefined:
```

Comment:
Chapters use the default icon that pass form the card opts but chapters does not have item type so it considered undefined.